chemicaltoolbox/rdkit/conformers.xml

```diff
@@ -0,0 +1,83 @@
+        <requirement type="package" version="@VERSION@">rdkit</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python3 '$__tool_directory__/conformers.py'
+        -s '$smiles_input'
+        -n '$num'
+        -a '$attempts'
+        -r '$rmsd'
+        -c '$conditional_cluster.cluster_method_opts'
+        #if $conditional_cluster.cluster_method_opts != 'none':
+            -t '$conditional_cluster.threshold'
+        #end if
+        -e '$emin'
+        -f '$conf_output'
+    ]]></command>
+    <inputs>
+        <param name="smiles_input" type="text" label="SMILES Format" help="ABC"/>
+        <param name="num" type="integer" label="Number of conformers to generate" value="1" help="XYZ"/>
+        <param name="attempts" type="integer" label="Number of attempts" value="0" help="LMO"/>
+        <param name="rmsd" type="float" label="Prune RMSD threshold" value="1.0" help="PQR"/>
+        <conditional name="conditional_cluster">
+            <param name="cluster_method_opts" type="select" label="Cluster" help="RST">
+                <option value="none" selected="true">None</option>
+                <option value="rmsd">RSDM</option>
+                <option value="tfd">TFD</option>
+            </param>
+            <when value="none">
+            </when>
+            <when value="rmsd">
+                <param name="threshold" type="float" label="Cluster threshold (default of 2.0 for RMSD and 0.3 for TFD)" value="2.0" help="TUV"/>    
+            </when>
+            <when value="tfd">
+                <param name="threshold" type="float" label="Cluster threshold (default of 2.0 for RMSD and 0.3 for TFD)" value="0.3" help="HGB"/>
+            </when>
+        </conditional>
+        <param name="emin" type="integer" label="Energy minimisation iterations (default of 0 means none)" value="0" help="KLQ"/>
+    </inputs>
+    <outputs>
+        <data name="conf_output" format="sdf" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="smiles_input" value="C(C1C(C(C(C(O1)O)O)O)O)O"/>
+            <param name="num" value="1"/>
+            <param name="attempts" value="0"/>
+            <param name="rmsd" value="1.0"/>
+            <param name="cluster_method_opts" value="rmsd"/>
+            <param name="emin" value="0"/>
+            <output name="conf_output" ftype='sdf' file="conf_output_rmsd.sdf"/>
+        </test>
+        <test>
+            <param name="smiles_input" value="C(C1C(C(C(C(O1)O)O)O)O)O"/>
+            <param name="num" value="1"/>
+            <param name="attempts" value="0"/>
+            <param name="rmsd" value="1.0"/>
+            <param name="cluster_method_opts" value="rmsd"/>
+            <param name="emin" value="0"/>
+            <output name="conf_output" ftype='sdf' file="conf_output_tfd.sdf"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        ABCCDEFGHIJKLMNOPQRSTUVWXYZ
+    ]]></help>
+    <citations>
+        <citation type="bibtex">
```

Comment:
can you also add the RDKit citation please? you can copy it from the descriptor tool